src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/VanillaItem.java

```diff
@@ -9,5 +9,6 @@
 	
 	public VanillaItem(Category category, ItemStack item, String name, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, name, recipeType, recipe);
+		setVanilla(true);
```

Comment:
Why simply delete something interesting instead of providing an option to toggle that?
I personally like the ability to change vanilla items recipes : it make chain armor obtainable and elytras mor difficult to get ^^. But there is some people that don't like that. So I make it optional.

It seems you didn't understand how the code works. Well, I know this is a kind of a mess there, that's why I'll do what I said in the review above. However, the armor was removed only when I set `CHAIN_HELMET.ignore-slimefun-modifications` to true. Otherwise there was still the armor in Slimefun ^^